quesma/clickhouse/table_discovery.go

```diff
@@ -275,6 +277,17 @@ func (sl *tableDiscovery) TableDefinitions() *TableMap {
 	return sl.tableDefinitions.Load()
 }
 
+func (sl *tableDiscovery) verify(tables map[string]discoveredTable) {
+	for _, table := range tables {
+		logger.Info().Msgf("verifying table %s", table.name)
```

Comment:
We should keep verification results in the table struct. We'll be able to show the "Problems" tab in the UI.

Reply:
Sounds like a great idea :)